### ISSUE TITLE
[dev-v5] fix: Select items with keystrokes in `FluentSelect`

### ISSUE THREAD

**Issue by MarvinKlein1508** [microsoft/fluentui-blazor#4768]
# 🐛 Bug Report

In v5 it is no longer possible to jump within the list by hitting any key. This makes it harder to navigate in the list if it contains more items.

## 💻 Repro or Code Sample
https://fluentui-blazor-v5.azurewebsites.net/Lists/Select

1. Click on the list
2. Press G to jump to green
3. Nothing is happening

## 🤔 Expected Behavior

https://github.com/user-attachments/assets/93475784-d34e-4cc2-9a81-47b8ae06959f

## 😯 Current Behavior

https://github.com/user-attachments/assets/b06ec29e-1c82-4ea8-a5cd-0decd96f3342

## 🌍 Your Environment

* OS & Device: Windows 11 64 Bit
* Browser latest Firefox, Chrome & Edge
* latest dev-v5 branch

**Comment by MarvinKlein1508** [comment]
This doesn't work in the underlying WC as well. I opened an issue there. https://github.com/microsoft/fluentui/issues/36122

**Comment by MarvinKlein1508** [comment]
@dvoituron this has been fixed in the underlying WC. An update for the packages should be all we need.

**Comment by MarvinKlein1508** [comment]
See https://github.com/microsoft/fluentui/pull/36232 for more details

**Comment by MarvinKlein1508** [comment]
This has been fixed in the underlying web components. But it is not working yet in our wrappers.

**Comment by MarvinKlein1508** [comment]
This has been fixed by updating the underlying WC. My mistake was I haven't cleared the cache.
